diff.py

```diff
@@ -1367,6 +1369,22 @@ def _normalize_adrp_differences(self, mnemonic: str, row: str) -> str:
 
         return row
 
+class DifferenceNormalizerARM32(DifferenceNormalizer):
+    def __init__(self, config: Config) -> None:
+        super().__init__(config)
+
+    def _normalize_arch_specific(self, mnemonic: str, row: str) -> str:
+        if self.config.ignore_addr_diffs:
+            row = self._normalize_bl(mnemonic, row)
```

Comment:
(would be nice to generalize this in the future to cover all archs, the logic should be the same for `jal` on mips for instance. just hasn't been a priority because for mips we almost solely use -o for which --ignore-addr-diffs isn't needed)

Reply:
Gotcha, makes sense.